runtime/hydradx/src/assets.rs

```diff
@@ -1583,6 +1584,46 @@ impl pallet_referrals::Config for Runtime {
 	type BenchmarkHelper = ReferralsBenchmarkHelper;
 }
 
+parameter_types! {
+	pub const LiquidationGasLimit: u64 = 4_000_000;
+}
+
+pub struct DummyEvm;
```

Comment:
is it possible to annotate with features runtime-benchmarks ?

Reply:
done